### PR TITLE
Assignments for Lessons

### PR DESCRIPTION
**Pull request by Headieh**
see different commits for different assignments

### CODE REVIEW

**Comment by Headieh** [comment]
ignore: 822d36f commit
use: dacca86 commit instead 
(just renamed file for assignment)

File: lesson02/calculator.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+//Lesson 2 Exercise 1
+
+let h = 36;
+const i = '12';
```
Comment (MadEste) [inline lesson02/calculator.js]: Nice work experimenting with const and strings for math.

File: lesson02/countdown.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+//Lesson 2 Exercise 2
+
+let x = 5;
+while (x > 0) { console.log(x-1); x--;
+}
```
Comment (MadEste) [inline lesson02/countdown.js]: 👍 Good job, also make sure you understand how to accomplish the same with a for loop syntax.

File: lesson02/canDrive.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const firstName = "Billy"
+const lastName = "Idol"
+let age = 64 //12
+const driveAge = 16
```
Comment (MadEste) [inline lesson02/canDrive.js]: good use of const instead of let!

File: lesson02/canDrive.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+if (age >= driveAge) {
+	console.log(`${firstName} ${lastName} can drive!`);
+	console.log(`Driven for: ${age - driveAge} years`);
+}
+
+if (age < driveAge) {
```
Comment (MadEste) [inline lesson02/canDrive.js]: Can also chain this with the first if statement and use `} else {`

File: lesson02/cardDeck.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+let suits = ["spades", "diamonds", "clubs", "hearts"];
+let values = ["A", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K"];
+
+for (let i = 0; i < values.length; i++) {
+	for (let j = 0; j < suits.length; j++) {
+		console.log(`${values[i]} ${suits[j]}`);
+	}
```
Comment (MadEste) [inline lesson02/cardDeck.js]: This works well, also good use of JS Template literals.

File: lesson03/exercise-one.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+//Do a console log after each item
+
+//Create an array, with at least 5 strings
+let food = ['grains', 'vegetables', 'fruit', 'protein'];
```
Comment (MadEste) [inline lesson03/exercise-one.js]: Only 4 elements were added, but I wont deduct for that.

File: lesson03/exercise-one.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+console.log(food);
+//Add an element to the end of the end of the array
+food.push('dairy');
+console.log(food);
+//Remove the third element
+food.splice(2, 1);
+console.log(food);
+//Create a string from the elements and comma separate them
+const foodList = food.join(',');
```
Comment (MadEste) [inline lesson03/exercise-one.js]: Nice work!

File: lesson03/exercise-two.js [hunk 8]
```diff
@@ -0,0 +1,5 @@
+//Take a sentence of at least five words and a period.
+const statement = 'Fruits are my favorite foods!';
+//Make an array from that sentence and console.log the third word.
+const statementArr = statement.split(' ');
+console.log(statementArr[2]);
```
Comment (MadEste) [inline lesson03/exercise-two.js]: 👍

File: lesson03/exercise-three.js [hunk 9]
```diff
@@ -0,0 +1,7 @@
+//Use a for loop to console log all of the elements plus the index and a comma
+
+let food = ['grains', 'vegetables', 'fruit', 'protein'];
+
+for (let i = 0; i < food.length; i++) {
+	console.log(`${i},${food[i]}`);
```
Comment (MadEste) [inline lesson03/exercise-three.js]: 👍 Perfect! Nice use of template literals

File: lesson04/exercise-one.js [hunk 10]
```diff
@@ -0,0 +1,14 @@
+function square() {
+	let input = prompt('Enter a number');
+	let result = 0;
+	result = Number(input) ** 2;
```
Comment (MadEste) [inline lesson04/exercise-one.js]: Nice work using the new ** operator!

can also combine lines 3 and 4

File: lesson04/exercise-one.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+	alert(result);
+}
+
+function multiply() {
+	let input1 = prompt('Enter 1st number');
+	let input2 = prompt('Enter 2nd number');
+	let result = 0;
+	result = Number(input1) * Number(input2);
```
Comment (MadEste) [inline lesson04/exercise-one.js]: 👍

File: lesson05/main.js [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+
+function bank() {
+	let menu = [];
+	const Q = 'Enter Q to quit';
+	const W = 'Enter W to withdraw';
+	const D = 'Enter D to deposit';
+	const B = 'Enter B to view balance';
+	menu.push(Q, W, D, B);
+	let input = prompt(menu.join("\n\n"));
+	switch (input.toLowerCase()) {
```
Comment (MadEste) [inline lesson05/main.js]: 👍

File: lesson05/main.js [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+let total = 0;
+
+function bank() {
+	let menu = [];
+	const Q = 'Enter Q to quit';
+	const W = 'Enter W to withdraw';
+	const D = 'Enter D to deposit';
+	const B = 'Enter B to view balance';
+	menu.push(Q, W, D, B);
+	let input = prompt(menu.join("\n\n"));
```
Comment (MadEste) [inline lesson05/main.js]: Nice!

**Pull request closed without merging**